sdk/keyvault/ci.yml

```diff
@@ -34,6 +34,8 @@ stages:
     parameters:
       ServiceDirectory: keyvault
       CtestRegex: azure-security-keyvault
+      LiveTestCtestRegex: live-azure-security-keyvault
+      SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
```

Comment:
Do we need to use the canary subscription specifically or can we use the normal subscription `$(sub-config-azure-cloud-test-resources)` ?

Reply:
I copied from storage.  I don't really know what should I use :(

Reply:
By default use `$(sub-config-azure-cloud-test-resources)` as it is our normal subscription. Canary and other subscriptions have different features enabled but `$(sub-config-azure-cloud-test-resources)` is a more pristine subscription.

Reply:
That is true assuming the canary features don't require any feature flags. I believe storage in particular needed feature flags so we kept them in the preview subscription where we are OK with feature flags.